mvp.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "An MVP for £500"
+layout: generic
```

Comment:
@iteles It's a jekyll thing. It just means the page on the website is using _layouts/generic.html.

And--mea culpa--I should have waited until today's meeting to make this pull request. I had meant to put the new copy live after the meeting this afternoon.